doc/data_format.rst

```diff
@@ -715,10 +715,16 @@ Offset     Length  Version  Description
   71+s      4       2-      Number of guest batches that can use the ride at the same time.
   75+s      4       2-      Maximum number of guests in each guest batch.
   79+s      4       2-      Duration of the ride's idle phase in milliseconds.
-  83+s      4       2-      Total duration of the ride's working phase in milliseconds.
-  87+s      4       1-      Text of the ride (reference to a TEXT block).
-  91+s                      Total length.
-=========  ======  =======  ===================================================================================
+  83+s      4       2-      Total duration of the ride's working phase per working cycle in milliseconds.
+  87+s      2       3-      Mimimum number of cycles (must be at least 1).
+  89+s      2       3-      Maximum number of cycles (must be at least the minimum number).
+  91+s      2       3-      Default number of cycles (must be at least the minimum and at most the maximum number).
+  93+s      2       3-      Maximum reliability factor (in percents of a percent).
+  95+s      2       3-      Daily reliability decrease (in percents of a percent).
+  97+s      2       3-      Monthly decrease of the maximum reliability (in percents of a percent).
```

Comment:
`in percents of a percent`? Interesting notion :)
I'd probably say `range [0..10000]`

I don't understand what "max reliability factor" is. Perhaps add a line of text explaining it.